examples/jax/encoder/test_model_parallel_encoder.py

```diff
@@ -503,6 +507,13 @@ def test_te_mxfp8_with_sp(self):
         actual = train_and_evaluate(self.args)
         assert actual[0] < 0.455 and actual[1] > 0.785
 
+    @unittest.skipIf(not is_bf16_supported(), "Device compute capability 8.0+ is required for BF16")
+    def test_te_bf16_shardy(self):
+        """Test Transformer Engine with BF16"""
+        self.args.enable_shardy = True
+        actual = train_and_evaluate(self.args)
+        assert actual[0] < 0.45 and actual[1] > 0.79
```

Comment:
```suggestion
        assert actual[0] < 0.45 and actual[1] > 0.79
```
```suggestion
        assert actual[0] < 0.455 and actual[1] > 0.785
```
So that it has the same tols as `test_te_bf16`. The test passes after relaxing the tols.